pcpostprocess/subtraction_plots.py

```diff
@@ -131,15 +164,166 @@ def do_subtraction_plot(fig, times, sweeps, before_currents, after_currents,
         subtracted_currents = before_currents[i, :] - before_leak_currents[i, :] - \
             (after_currents[i, :] - after_leak_currents[i, :])
         ax.plot(times*1e-3, subtracted_currents, label=f"sweep {sweep}", alpha=.5)
-
+        corrs, _ = pearsonr(subtracted_currents, voltages)
+        sweep_list += [sweep]
+        pcs += [corrs]
         #  Cycle to next colour
         ax.plot([np.nan], [np.nan], label=f"sweep {sweep}", alpha=.5)
-
+    # sortedy = sorted(subtracted_currents)
+    # ax.set_ylim(sortedy[30]*1.1, sortedy[-30]*1.1)
     ax.set_ylabel(r'$I_\mathrm{obs} - I_\mathrm{L}$ (mV)')
-    ax.set_xlabel('$t$ (s)')
+    ax.legend()
+    # ax.set_xlabel('$t$ (s)')
 
     long_protocol_ax.plot(times*1e-3, voltages, color='black')
     long_protocol_ax.set_xlabel('time (s)')
     long_protocol_ax.set_ylabel(r'$V_\mathrm{cmd}$ (mV)')
     long_protocol_ax.tick_params(axis='y', rotation=90)
 
+    corr_dict = {'sweeps': sweeps, 'pcs': pcs}
+    return corr_dict
+
```

Comment:
Duplicated code from leak_correct.py?

We could remove the other file, but I don't think it's been deleted by these commits